x/auth/keeper/grpc_query_test.go

```diff
@@ -236,7 +317,7 @@ func (suite *KeeperTestSuite) TestAddressBytesToString() {
 
 			ctx := sdk.WrapSDKContext(suite.ctx)
 
-			res, err := suite.queryClient.AddressBytesToString(context.Background(), tc.req)
+			res, err := suite.queryClient.AddressBytesToString(ctx, tc.req)
```

Comment:
Does it make a difference  either use `context.Background()`  or `ctx := sdk.WrapSDKContext(suite.ctx)` in my 2 tests  `TestAddressStringToBytes()` and  `TestAddressBytesToString` ? In other tests `ctx := sdk.WrapSDKContext(suite.ctx)` is widely used.